src/test/cpp/util/path_posix_test.cc

```diff
@@ -184,4 +184,52 @@ TEST(PathPosixTest, MakeAbsoluteAndResolveEnvvars) {
             JoinPath(GetCwd(), "%PATH%"));
 }
 
+TEST(PathPosixTest, NormalizeAbsPath) {
+  EXPECT_EQ(TestOnly_NormalizeAbsPath(""), "");
```

Comment:
Can you also a test for path starting with `../`, eg. `../x/y`

Reply:
Done.